src/main/java/org/myrobotlab/service/data/Pose.java

```diff
@@ -0,0 +1,22 @@
+package org.myrobotlab.service.data;
+
+public class Pose {
```

Comment:
there is already a Pose class in the kinematics package used by IK3d... why are we creating yet another Pose class?

Reply:
The Pose in IK has a lot of "stuff" not related to a instance of time Pose
Speed as an example
PoseMove ?   This doesn't seem to related to instance positional information

Reply:
This can be consolidated into a singular type hierarchy along with Point, Orientation, and Position. If a Position has a Point origin and an Orientation, then a Pose can have a Position and a time and name

Reply:
The original Pose class didn't have the junk that you mention.  Actually, in the commit history, @supertick  you were the one that added that stuff.  The Pose class as originally intended was to have the servo positions, the servo speeds , a name , and some metadata about when it was created.  The main point of the Pose class was so that it could serialize into the pose.json...

Reply:
alright .. I'll normalize them

Reply:
If this class if fundamentally something different, we should name it something other than Pose because we already have a Pose class..

Reply:
its not, it should be reconciled/normalized

Reply:
on closer inspection this is fundamentally different ...
The Pose you constructed is a set of positions of servo rotations...
This Pose I'm trying to describe is 3D position and orientation of some point - in the case where its being used its describing the position and orientation of :
* the headset
* the left controller
* the right controller
its more than a 3d point because it has orientation